src/Discord.Net.Commands/Builders/ModuleBuilder.cs

```diff
@@ -25,6 +26,8 @@ public class ModuleBuilder
         public IReadOnlyList<Attribute> Attributes => _attributes;
         public IReadOnlyList<string> Aliases => _aliases;
 
+        internal Optional<TypeInfo> TypeInfo { get; set; }
```

Comment:
Isn't TypeInfo a class? Surely we could compare against null rather than bringing in an extra layer of indirection?